server-spi-private/src/main/java/org/keycloak/component/ComponentModelScope.java

```diff
@@ -84,7 +84,8 @@ public Integer getInt(String key) {
     @Override
     public Integer getInt(String key, Integer defaultValue) {
         final String res = componentConfig.get(prefix + key, null);
-        return (res == null) ? origScope.getInt(key, defaultValue) : Integer.parseInt(res);
+        if (res == null) return origScope.getInt(key, defaultValue);
+        return Integer.parseInt(res);
```

Comment:
Why is this change here?

Reply:
I am not sure why but without throws NPE:
```
Caused by: java.lang.NullPointerException
	at org.keycloak.component.ComponentModelScope.getInt(ComponentModelScope.java:87)
	at org.keycloak.component.ComponentModelScope.getInt(ComponentModelScope.java:81)
	at org.keycloak.models.map.storage.jpa.client.JpaClientMapStorageProviderFactory.lazyInit(JpaClientMapStorageProviderFactory.java:147)
```

Reply:
https://github.com/keycloak/keycloak/pull/9139